src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
+    white,
+    red,
+    blue
```

Comment:
Enum constant should be written as well as final fields with upper case
```suggestion
    BLACK,
    WHITE,
    RED,
    BLUE
```